packages/invoices/src/OutboundShipment/api/api.ts

```diff
@@ -94,6 +94,11 @@ const outboundParsers = {
       invoiceId: line.invoiceId,
       numberOfPacks: line.numberOfPacks,
       stockLineId: line.stockLine?.id ?? '',
+      tax: { percentage: 10 },
```

Comment:
Added this for now as we're not able to query for tax values - pretending they don't exist. Also the totals need to be sent rather than calculated on the backend, which I think is not ideal. Ideally would just send the `sellPrice` for outbounds (and the tax should generally always be taken from somewhere centrally, with editable tax on the line being the edge case  🤷

Reply:
I thought so too - wasn't following the reasoning on why this couldn't be calculated in the api